features/facebookImport/src/components/scrollableXGradient/scrollableXGradient.css

```diff
@@ -0,0 +1,39 @@
+.startScroll .scrollable {
+    position: relative;
+    overflow: hidden;
+    display: flex;
+    flex-direction: row;
+
+
+}
+
+.startScroll h1 {
+font-size: 20px;
+margin: 4px 0px !important;
```

Comment:
Shouldn't need !important here. Try not to use it

Reply:
wanted to get rid of this anyway :D